Helper/Data.php

```diff
@@ -1796,4 +1797,25 @@ public function getPspReferenceSearchUrl($pspReference, $liveEnvironment)
             $pspReference
         );
     }
+
+    /**
+     * Get pspReference only if there are additional string attached
+     * @param $pspReference
+     */
+    public function getPspReferenceWithNoAdditions($pspReference)
+    {
+        if ($pspReference == '') {
```

Comment:
Maybe this is better
```suggestion
        if (empty($pspReference)) {
```